README.md

```diff
@@ -32,6 +32,7 @@ The native iOS SDK for Tenjin. To learn more about Tenjin and our product offeri
   - [Tenjin + IronSource Impression Level Ad Revenue Integration](#ironsource)
   - [Tenjin + HyperBid Impression Level Ad Revenue Integration](#hyperbid)
   - [Tenjin + AdMob Impression Level Ad Revenue Integration](#admob)
+  - [Tenjin + TopOn Impression Level Ad Revenue Integration](#topon)
```

Comment:
@IrminDzevlan 
Please change the PR 
there are some changes proposed in #104 
I request you to use base the PR according to the changes in #104 

TODO

- [ ] Base the PR using the PR branch from #104 
- [ ] Remove `Tenjin +` from Topon
- [ ] Update Release notes with Topon and also add version

Reply:
@IrminDzevlan the PR #104 is merged to master. Please can you resolve the conflicts and also review the above TODO suggestions for this PR.